### ISSUE TITLE
Support Veeam version v11.0.1.1261 for backup and recovery

### ISSUE THREAD

**Issue by harikrishna-patnala** [apache/cloudstack#6554]
##### ISSUE TYPE

 * Improvement Request
##### COMPONENT NAME

~~~
Backup and recovery
~~~

##### SUMMARY

Veeam plugin is already supported and the documentation is available here http://docs.cloudstack.apache.org/en/latest/adminguide/veeam_plugin.html

Since the version v11.0.1.1261 the powershell commands has changed and few commands are no more valid.

Examples:
1. "PowerShell Add-PSSnapin VeeamPSSnapin" This command is not required anymore. Instead, all commands can be run as "PowerShell -command "<ActualCommand>""
2.                 String.format("$Job = Get-VBRJob -name \"%s\"", backupName),
                "$Job.GetBackupTargetRepository() ^| select Name | Format-List"
In this command "^" is not valid.

##### STEPS TO REPRODUCE

~~~
1. Setup Veeam with version v11.0.1.1261
2. Integrate with CloudStack
3. Wait for the backup sync to happend.
~~~

##### EXPECTED RESULTS

~~~
Using Veeam v11.0.1.1261 with CloudStack should work seemlessly
~~~

##### ACTUAL RESULTS

~~~
Using Veeam v11.0.1.1261 with CloudStack failed
~~~

**Comment by harikrishna-patnala** [comment]
@SadiJr let's continue our discussion here (from PR#6555) since that PR is merged and closed.

"@harikrishna-patnala sorry for the delay in answer. I tested in my lab with the Veeam build 11.0.1.1261 P20211211, comparing with the main commit https://github.com/apache/cloudstack/commit/bca60761c502f1fa04c1d462128ef537c0e7ce73, and the error reported in issue https://github.com/apache/cloudstack/issues/6465 is fixed. About issue https://github.com/apache/cloudstack/issues/6554, I cannot reproduce the second example; can you please inform your version of Windows and PowerShell?"

>> I'm using Windows server 2019 and with powershell version "5.1.17763.771", May I know your's too where you said you were able to use the veeam version 11.0.1.1261

**Comment by weizhouapache** [comment]
addressed by #8241

**Comment by rohityadavcloud** [comment]
This is addressed in https://github.com/apache/cloudstack/pull/8241 closing
